### PR TITLE
Add `Cancel` button to location input form.

### PR DESCRIPTION
**Pull request by bdruth**
* Create cancel sequence
  * resets form (fixed!)
  * navigates back using history
* Create humble object for browser to test `history.back()`

Addresses AC in #170 

New cancel button:
image

### CODE REVIEW

File: src/presenter/cerebral/actions/goBackAction.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const goBackAction = ({ humbleWindow }) => {
```
Comment (kkoskelin) [inline src/presenter/cerebral/actions/goBackAction.js:1]: Thanks for not shadowing `window`. 😄

File: src/presenter/cerebral/actions/goBackAction.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { goBackAction } from './goBackAction';
+import { presenter } from '../../../presenter/presenter';
+import { runAction } from 'cerebral/test';
+
+describe('goBackAction', () => {
+  it('should call `back` on history', () => {
+    const mockWindow = {
+      history: () => {
+        return {
+          back: () => {
```
Comment (kkoskelin) [inline src/presenter/cerebral/actions/goBackAction.test.js:10]: I know I'm reviewing after a merge, but ... I would like to suggest 
`const backMock = jest.fn();` 
and then 
`back: backMock,`.
Then you can discard the `expect.assertions`, call `runAction()` and then 
`expect(backMock).toHaveBeenCalled()`

File: src/presenter/cerebral/frameworks/humbleBrowser.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+const humbleWindow = {
+  history: () => {
+    return history;
```
Comment (kkoskelin) [inline src/presenter/cerebral/frameworks/humbleBrowser.js:3]: Are you referencing the global `history` deliberately? If so, I would recommend calling it out as `window.history`.

File: src/presenter/cerebral/actions/resetFormAction.test.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  it('should reset form data', async () => {
+    const initialState = {
+      state: {
+        form: {
+          ...defaultForm(),
+          city: 'Testing',
+        },
+      },
+    };
+    return runAction(resetFormAction, initialState).then(({ state }) =>
```
Comment (kkoskelin) [inline src/presenter/cerebral/actions/resetFormAction.test.js:16]: I would recommend not mixing `async` with Promise calls like `.then`.  Rather, I would prefer to see 
`const { state } = await runAction(...);` followed by the assertion.

**Review by kkoskelin — COMMENTED** [review]
image
I also would recommend addressing the react warnings in the linting regarding loop keys.

File: src/presenter/cerebral/actions/goBackAction.js (hunk 1 above)
Comment (bdruth) [reply src/presenter/cerebral/actions/goBackAction.js:1]: lol, I definitely was at one point ... ;)

File: src/presenter/cerebral/actions/goBackAction.test.js (hunk 2 above)
Comment (bdruth) [reply src/presenter/cerebral/actions/goBackAction.test.js:10]: That's awesome, I'll try that! Thank you.

File: src/presenter/cerebral/actions/resetFormAction.test.js (hunk 4 above)
Comment (bdruth) [reply src/presenter/cerebral/actions/resetFormAction.test.js:16]: this is what happens when I copy/paste from SO :P

**Pull request merged**